promql/promqltest/testdata/functions.test

```diff
@@ -1452,3 +1452,22 @@ load 1m
 
 eval range from 0 to 5m step 1m round(mixed_metric)
 	{} _ 1 2 3
+
+clear
+
+# check vanish
+load 1m
+	metric{a="1"} 0x100 _x100 0x100
+	metric{a="2"} 0x100 _x100 _x100
+
+eval instant at 100m vanish(metric[30m])
```

Comment:
This can be replaced by `present_over_time(metric[30m])-1 and metric or present_over_time(metric[30m])` (and correspondingly below). It might be a bit wordy, but we are also addressing a niche use case here.

Furthermore, I don't think that a function returning 0 or 1 is good for alerting. If you want to alert on disappearing time series, you want an expression that contains exactly the disappearing series and nothing else, something like `present_over_time(some_metric[1h]) unless present_over_time`, see also my comment on the proposal.